.github/workflows/bump-license-year-in-all-projects.yml

```diff
@@ -0,0 +1,45 @@
+#############################################################################
+# GitHub Action to bump license year in all projects
+#
+# Workflow starts every new year.
+#
+#############################################################################
+name: "Bump license year in all project"
+on:
+  schedule:
+    - cron: "0 0 1 1 *"
+  workflow_dispatch:
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  bump:
+    name: Bump license year in all projects
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set Current Year
+        id: CURRENT_YEAR
+        run: |
+          echo "::set-output name=year::$(date +'%Y')"
+      - name: Modify File
+        run: |
+          ./.ci/bump-license-year-in-all-projects.sh $(expr ${{ env.YEAR }} - 1) ${{ env.YEAR }} .
+        env:
+          YEAR: ${{ steps.CURRENT_YEAR.outputs.year }}
+      - name: Create Pull Request
```

Comment:
onbody will review this PR, it is very technical update, lets just push code to master.

Reply:
@romani How I can proceed with this PR then?

Reply:
github action should just update code and push code changes to master without PR.

we will need this ability when we do "mvn set:version"  in release process.

Please plan with it in your fork and to test it.